specs/vm/instruction_set.md

```diff
@@ -1417,6 +1418,22 @@ This modifies the `balanceRoot` field of the appropriate output.
 `messageID` is added to the `OutputMessage` Merkle tree as part of block header.
 TODO: document output messages merkle tree construction and maintenance and link here
 
+### SCWQ: State clear sequential 32 byte slots
+
+|             |                                                                               |
+|-------------|-------------------------------------------------------------------------------|
+| Description | A sequential series of 32 bytes is cleared from the current contract's state. |
+| Operation   | ```STATE[MEM[$rA, 32], 32  * $rB] = None;```                                        |
+| Syntax      | `scwq $rA, $rB`                                                               |
+| Encoding    | `0x00 rA rB - -`                                                              |
+| Notes       |                                                                               |
+
+Panic if:
+
+- `$rA + 32 * $rB` overflows
```

Comment:
Wait, you don't need to multiple by `$rB` here. `MEM[$rA, 32]` holds the 32 byte slot index and that's it.

Reply:
I made the same comment here: https://github.com/FuelLabs/fuel-specs/pull/415#discussion_r993929556

Reply:
fixed